tests/github/fair_software/badge/test_checker_with_noreason_user_config.py

```diff
@@ -0,0 +1,123 @@
+    def test_check_checklist(self, mocker: Mocker, user_config, capsys):
+        with mocker:
+            checker = get_mocked_checker(user_config)
+            assert checker.check_checklist() is True
+            captured = capsys.readouterr()
+            assert "no reason provided" in captured.out
+
+    def test_check_citation(self, mocker: Mocker, user_config, capsys):
+        with mocker:
+            checker = get_mocked_checker(user_config)
+            assert checker.check_citation() is True
+            captured = capsys.readouterr()
+            assert "no reason provided" in captured.out
+
+    def test_check_license(self, mocker: Mocker, user_config, capsys):
+        with mocker:
+            checker = get_mocked_checker(user_config)
+            assert checker.check_license() is True
+            captured = capsys.readouterr()
+            assert "no reason provided" in captured.out
+
+    def test_check_registry(self, mocker: Mocker, user_config, capsys):
+        with mocker:
+            checker = get_mocked_checker(user_config)
+            assert checker.check_registry() is True
+            captured = capsys.readouterr()
+            assert "no reason provided" in captured.out
+
+    def test_check_repository(self, mocker: Mocker, user_config, capsys):
+        with mocker:
+            checker = get_mocked_checker(user_config)
+            assert checker.check_repository() is True
+            captured = capsys.readouterr()
+            assert "no reason provided" in captured.out
+
+    def test_compliance(self, mocker: Mocker):
+        pass
+
+    def test_has_ascl_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_bintray_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_citation_file(self, mocker: Mocker):
+        pass
+
+    def test_has_citationcff_file(self, mocker: Mocker):
+        pass
+
+    def test_has_codemeta_file(self, mocker: Mocker):
+        pass
+
+    def test_has_conda_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_core_infrastructures_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_cran_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_crates_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_license(self, mocker: Mocker):
+        pass
+
+    def test_has_maven_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_npm_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_open_repository(self, mocker: Mocker):
+        pass
+
+    def test_has_pypi_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_rsd_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_zenodo_badge(self, mocker: Mocker):
+        pass
+
+    def test_has_zenodo_metadata_file(self, mocker: Mocker):
+        pass
+
+    def test_is_on_github_marketplace(self, mocker: Mocker):
+        pass
+
+    def test_readme(self, mocker: Mocker):
+        pass
+
+    def test_repo(self, mocker: Mocker):
+        pass
```

Comment:
the low-level check functions are not implemented. Was this intentional?

Reply:
lazyness, they have now been copied over

Reply:
spawned #227